File: Python/02. Simple Operations and Calculations/Exercise/Tailoring Workshop.py
tables = int(input())
length = float(input())
width = float(input())

tableclothArea = (length + 0.60) * (width + 0.60)
tablesquareArea = (length / 2) * (length / 2)
tableclothPrice = tableclothArea * 7 * tables
tablesquarePrice = tablesquareArea * 9 * tables
USD = tableclothPrice + tablesquarePrice
BGN = USD * 1.85

print(f"{USD:.2f} USD")
print(f"{BGN:.2f} BGN")

File: C#/02. Simple Operations and Calculations/Exercise/Tailoring Workshop.cs
using System;

namespace Tailoring_Workshop
{
    class Program
    {
        static void Main(string[] args)
        {
            int tables = int.Parse(Console.ReadLine());
            double length = double.Parse(Console.ReadLine());
            double width = double.Parse(Console.ReadLine());

            double tableclothPrice = (length + 0.30 * 2) * (width + 0.30 * 2) * 7;
            double tablesquarePrice = (length * length / 4) * 9;

            double USD = tableclothPrice + tablesquarePrice;
            double BGN = USD * 1.85;

            Console.WriteLine($"{USD * tables:F2} USD");
            Console.WriteLine($"{BGN * tables:F2} BGN");
        }
    }
}
